index.bs

```diff
@@ -240,6 +245,9 @@ interface ImageCapture {
 
   <dt><dfn dict-member for="PhotoSettings"><code>fillLightMode</code></dfn></dt>
   <dd>This reflects the desired <a>fill light mode</a> (flash) setting.</dd>
+
+  <dt><dfn dict-member for="PhotoSettings"><code>imageFormats</code></dfn></dt>
+  <dd>This reflects the desired <a>image format</a> for the captured photo. If the UA supports desired <a>image format</a>, the {{Blob/type!!attribute}} of the {{Blob}} produced by <a>takePhoto()</a> MUST be of the {{imageFormats!!dict-member}} [=image format|type=].</dd>
```

Comment:
> s/MUST be of/MUST be one of/
s/{{imageFormats!!dict-member}}/{{PhotoCapabilities/imageFormats}}/ ?

Fixed, however, I would prefer original version. If web developer explicitly requests e.g., "image/tiff", proper type should be returned, or takePhoto() should fail if unsupported type was requested. Now, UA is allowed to return whatever it supports, regardless of provided PhotoSettings.

> Also, despite [= =] syntax being valid, this document uses mostly the likes of <a>image format</a>.

Fixed. Is there a way to override display text for anchors? I use [==] in other specs that use bikeshed, as it is easier to make plurals or tailor text within different contexts.